lib/gc/Transforms/GPU/LinalgToXeGPU.cpp

```diff
@@ -83,6 +83,12 @@ struct TilesArray {
   SmallVector<SmallVector<Value>> tileMatrix;
 };
 
+static xegpu::TensorDescType getTensorDescType(llvm::ArrayRef<int64_t> shape,
```

Comment:
The IMEX has originally a single function like:

```c++
TensorDescType get(ArrayRef<int64_t> shape, Type type, int aaa = 1, bool bbb = true);
```

And they added another similar overload function:

```c++
TensorDescType get(ArrayRef<int64_t> shape, Type type, int aaa = 1, SomeOtherType bbb = {});
```

This makes the default arguments unusable, `xegpu::TensorDescType::get(shape, elementType)` will end up with ambitious overloading error.